packages/clerk-js/src/ui/components/UserProfile/ConnectedAccountsPage.tsx

```diff
@@ -75,8 +75,9 @@ const AddConnectedAccount = () => {
       })
       .then(res => {
         if (res.verification?.externalVerificationRedirectURL) {
-          return navigate(res.verification.externalVerificationRedirectURL);
+          return navigate(res.verification.externalVerificationRedirectURL.href);
         }
+        return;
```

Comment:
🙃 is this required?

Reply:
We get `Not all code paths return a value` and can't build. I can do `void navigate()` instead.

Reply:
that's okay. leave it as is.